css/DesktopVersion.css

```diff
@@ -0,0 +1,417 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;1,100;1,200;1,300;1,400&display=swap");
+
+* {
+  font-family: "Poppins", sans-serif;
+}
+
+@media only screen and (min-width: 768px),
+  (min-width: 1024),
+  (min-width: 1440px) {
+  body {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    background-color: #dfe1e6;
+  }
+
+  header {
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100%;
+    background-color: white;
+    color: black;
+    padding: 10px;
+  }
```

Comment:
- [ ] Good work building your Desktop header ✔️. However, please kindly ensure to fix the margins in your design to stick as much as possible to the Figma design as requested by the previous reviewer. 🤓

This is the Figma design 👇🏽

![figma](https://user-images.githubusercontent.com/116701485/236356152-3dc15d9b-2968-4545-92ad-a5c4ed0549b3.png)

This is your design 👇🏽

![design](https://user-images.githubusercontent.com/116701485/236356138-5fcfbf01-5a42-41f7-8f69-bf0f3094e740.png)